server/pbench/bin/pbench-unpack-tarballs

```diff
@@ -222,27 +223,22 @@ while read size result ;do
     mkdir -p $RESULTS/$hostname/$prefix
     status=$?
     if [[ $status -ne 0 ]] ;then
-        echo "$TS: prefix failure - code $status:         mkdir -p $RESULTS/$hostname/$prefix" >&4
+        echo "$TS: code $status: mkdir -p $RESULTS/$hostname/$prefix" >&4
     else
```

Comment:
Please add a comment, explaining what's happening in the following section of code.